pkg/manager/manager.go

```
@@ -781,8 +1041,490 @@
+	if sr.Unshareable {
+		fmt.Printf("Skipping unshareable rule: %s\n", sr.Key)
+		return nil
+	}
+
+	// Handle conflict resolution if needed
+	key := sr.Key
+	if existingRules[key] {
+		var action string
+		var err error
+		key, action, err = resolveConflict(key, autoResolve)
+		if err == ErrSkipRule {
```

golangci-lint (1.64.8)

comparing with == will fail on wrapped errors. Use errors.Is to check for a specific error

(errorlint)